autoload/slime.vim

```diff
@@ -63,6 +63,28 @@ function! s:KittyConfig() abort
   let b:slime_config["listen_on"] = input("kitty listen on: ", b:slime_config["listen_on"])
 endfunction
 
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+" Wezterm
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+function! s:WeztermSend(config, text)
+	call system("wezterm cli send-text --no-paste --pane-id " . shellescape(a:config["pane_id"]) . " " . shellescape(a:text))
```

Comment:
I don't know anything about slime other than what I quickly skimmed from its readme, but it suggests that `g:slime_bracketed_paste` should control whether `--no-paste` is passed here